FAQ.md

```diff
@@ -78,7 +78,9 @@ An example, `foo.com/product/index.html?id=204&page=2`, gray release based on `i
 1. Group A：id <= 1000
 2. Group B：id > 1000
 
-here is the way:
+There are two different ways to do this：
+
+1. Use the vars attribute of route to do it.
```

Comment:
`vars` is a field in the Route structure, please use backquotes to wrap it.

Reply:
updated.